test/api/traversing.js

```diff
@@ -680,6 +681,20 @@ describe('$(...)', function() {
     });
   });
 
+  if (typeof Symbol !== 'undefined') {
+    describe('[Symbol.iterator]', function() {
+
+      it('should yield each element', function() {
+        // The equivalent of: for (const element of $('li')) ...
+        var $li = $('li'),
+            iterator = $li[Symbol.iterator]();
+        expect(iterator.next().value.attribs['class']).to.equal('apple');
+        expect(iterator.next().value.attribs['class']).to.equal('orange');
+        expect(iterator.next().value.attribs['class']).to.equal('pear');
```

Comment:
Can you also assert that iteration terminates as expected?

Reply:
Absolutely, fixed in 1b80376066c8e0f272643857c5b5567ccb162aef